src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue21109.cs

```diff
@@ -12,7 +12,7 @@ public Issue21109(TestDevice device) : base(device) { }
 
 		public override string Issue => "[Android] MAUI 8.0.3 -> 8.0.6 regression: custom handler with key listener no longer works";
 
-		[Test, Order(1)]
+		[FlakyTest, Order(1)]
```

Comment:
Could create an issue to re-enable (fix it) and add a comment with the link here?